jmclient/jmclient/schedule.py

```diff
@@ -17,6 +19,8 @@
     the chance of success on re-trying
 """
 
+NO_ROUNDING = 16 #max btc significant figures not including LN
```

Comment:
Is this definitely schedule specific? I'd guess it fits better in `jmbitcoin`? Admittedly nbd, feel free to ignore.